src/metabase/middleware/util.clj

```diff
@@ -22,6 +22,11 @@
 
 (defn cacheable?
   "Can the ring request be permanently cached?"
-  [{:keys [uri query-string]}]
-  ;; match requests that are js/css and have a cache-busting query string
-  (and query-string (re-matches #"^/app/dist/.*\.(js|css)$" uri)))
+  [{:keys [request-method uri query-string], :as request}]
+  (and (= request-method :get)
```

Comment:
It would be nicer to change this to a property of the response itself, e.g. a key like`:metabase.api/cacheable?`. But that's out of the scope for this PR